doc/contributing/docs/api.rst

```diff
@@ -82,7 +88,7 @@ Configuration parameters
 For every configuration parameter, list the following details:
 
 *   Since which Tarantool version
-*   General description
+*   General description (in imperative mood)
```

Comment:
Minor: "imperative mood" seems unclear for parameter descriptions – they are mostly noun phrases.